src/controllers/simplified-account/settings/switch-psp/switch-to-stripe/switch-to-stripe.controller.ts

```diff
@@ -0,0 +1,84 @@
+import type { NextFunction } from 'express'
+import type { ServiceRequest, ServiceResponse } from '@utils/types/express'
+import { response } from '@utils/response'
+import StripeTasks from '@models/StripeTasks.class'
+import { getConnectorStripeAccountSetup } from '@services/stripe-details.service'
+import StripeAccountSetup from '@models/StripeAccountSetup.class'
+import TaskStatus from '@models/constants/task-status'
+import paths from '@root/paths'
+import GatewayAccountSwitchPaymentProviderRequest from '@models/gateway-account/GatewayAccountSwitchPaymentProviderRequest.class'
+import gatewayAccountsService from '@services/gateway-accounts.service'
+import formatPSPName from '@utils/format-PSP-name'
+import formatServiceAndAccountPathsFor from '@utils/simplified-account/format/format-service-and-account-paths-for'
+
+async function get(req: ServiceRequest, res: ServiceResponse) {
+  const account = req.account
+  const service = req.service
+  const stripeAccountSetup = (await getConnectorStripeAccountSetup(
+    service.externalId,
+    account.type
+  )) as StripeAccountSetup
```

Comment:
would it make more sense to just add the return type to `getConnectorStripeAccountSetup`?

Reply:
it's part of the connector client that i'm half way through converting to typescript so will soon be unnecessary but still required here for the time being